### PR TITLE
Overrides cleanup - part2

### PR DESCRIPTION
**Pull request by wojtek-t**
/assign @mm4tt

### CODE REVIEW

File: clusterloader2/testing/experimental/storage/pod-startup/suite.yaml [hunk 1]
```diff
@@ -45,3 +51,9 @@
   overridePaths:
     - testing/experimental/storage/pod-startup/volume-types/secret/override.yaml
     - testing/experimental/storage/pod-startup/max_volumes_per_node/override.yaml
+
+#- identifier: persistent-vol-per-node
```
Comment (mm4tt) [inline clusterloader2/testing/experimental/storage/pod-startup/suite.yaml:55]: Not sure if I follow why we're adding this comments. Could you explain?

Comment (wojtek-t) [reply clusterloader2/testing/experimental/storage/pod-startup/suite.yaml:55]: I want to clean those up at once - see: https://github.com/kubernetes/test-infra/pull/17727/files#diff-1db66d4d90bd960aa1cd23062e094030L2

**Pull request merged**